index.html

```diff
@@ -92,6 +92,11 @@ <h3>
             <i class="fa fa-mouse-pointer"></i>
             Click for Demo
         </button>
+        <br>
+        <button id="focus" class="btn">
```

Comment:
this should be hidden when currently on the focus URL

Reply:
Done!